libsrc/runtime/absvaludiv.s

```diff
@@ -0,0 +1,22 @@
+;
+; Christian Krueger, 23-May-2018
+;
+; CC65 runtime: helper call for mod/div with signed ints
+;
+; When negating values, we will ignore the possibility here, that one of the
+; values is $8000, in which case the negate will fail.
+
+        .export         absvaludiv
+        .import         _abs, popax, udiv16
+        .importzp       ptr1, ptr4
+
+
+absvaludiv:
```

Comment:
That name should be changed to `absvaludiv16` because someone might want to make an `absvaludiv32` function, in the future.